src/style.css

```diff
@@ -0,0 +1,144 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  display: block;
+  margin: 8px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+[data-qa="big"] {
+  width: 100%;
+  height: 70px;
+  margin: 20px auto 0;
+}
+
+.labelSearch {
+  position: relative;
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.labelSearch::after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.labelSearchBig::after {
+  background-image: url(../src/images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.inputTextBig {
+  width: 100%;
+  height: 100%;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+  font-family: inherit;
+  color: #3d4e61;
+  padding: 0 0 0 62px;
+  border-radius: 4px;
+}
+
+.inputTextBig:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  font-style: normal;
+}
+
+.inputTextBig:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.inputTextBig::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+  font-family: inherit;
+  color: #3d4e61;
+}
+
+[data-qa="small"] {
+  width: 100%;
+  height: 42px;
+  margin-top: 20px;
+}
+
+.labelSearchSmall::after {
+  background-image: url(../src/images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.inputTextSmall {
+  width: 100%;
+  height: 100%;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  font-size: 14px;
+  line-height: 19px;
+  font-weight: 300;
+  font-family: inherit;
+  padding: 0 0 0 33px;
+  border-radius: 4px;
```

Comment:
if you have two or more similar elements with portions of similar styles with different values - use one of the elements as the basic case, and override necessary styles for other cases. Explanation: The point is not in the names of the classes, the point is: when there are several similar elements, ex., 2 inputs, for one we can give a class input, for example, and for the second - input input--small. We write all the styles for .input, but for .input-small we write only those styles that differ in design, and we need this second input to look a little different. Element with class .input without extra classes should also look like a full-fledged styled element.

https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md